FOUNDATIONS.md

```diff
@@ -13,15 +13,12 @@ ensure the success of future improvements (especially type systems).
 
 ## Linked
 
-IPLD must support linking to any IPLD node (even if the node is in the middle of
-a block). That is, IPLD must support arbitrary IPLD paths in links.
+The IPLD Data Model includes Links. A Link can be resolved to reach another IPLD Node.
 
-**Motivation:** Considering this in the context of programming languages, not
-being able to *store* a pointer to a struct *inside* of another struct would be
-severely limiting.
-
-NOTE: We don't currently support arbitrary paths but, in the context of
-programming, we really need to.
+**Motivation:** Linking makes it possible to build data structures which are
+theoretically unbounded in size, while still being traversable, consistent,
```

Comment:
Unbounded? Not without cycles.

Reply:
How would you rather phrase this?

I thought we typically referred to the size of a merkle-dag as being "unbounded".  The size of a blockchain certainly isn't often called "bounded"!